group_project_v2/group_project.py

```diff
@@ -446,12 +445,23 @@ def grade_display_stages(self):
         return self.get_children_by_category(GradeDisplayStage.CATEGORY)
 
     @property
-    def max_grade_display_date(self):
+    def get_grade_display_stage(self):
```

Comment:
@ziafazal nit: bad name - `get` is a verb, so it should be a method, not a property